dpgen/dispatcher/Dispatcher.py

```diff
@@ -92,11 +103,21 @@ def run_jobs(self,
                                        forward_task_deference,
                                        outlog,
                                        errlog)
-        while not self.all_finished(job_handler, mark_failure) :
-            time.sleep(60)
-        # delete path map file when job finish
-        # _pmap.delete()
+        finished_job, all_job = self.all_finished(job_handler, mark_failure)
+        cnt = 0
+        while finished_job != all_job:
+            finished_ratio = float(finished_job) / float(all_job)
+            if finished_ratio >= async_check:
+                if cnt >= 10:
```

Comment:
The reason for introducing `cnt` and `multiprocessing` here should be explained within comments.

Reply:
I'm trying to understand how the async mode is achieved.  So if the condition `finished_ratio >= async_check`  is satisfied,  you will check if there is more tasks finished and try to collect them for ten times?  Is `multiprocessing` necessary?  Sometimes `multiprocessing` on my Mac is not stable, so I would avoid using it unless necessary.